sdk/notificationhubs/notification-hubs/CHANGELOG.md

```diff
@@ -8,6 +8,9 @@
 
 ### Bugs Fixed
 
+- Added top level `browser` and `react-native` support in the `package.json` and a common implementation for WebCrypto for both.
```

Comment:
nit: put this entry under other changes, or new features?

Reply:
Moved to other changes